test-e2e/specs/presenceSpec.js

```diff
@@ -0,0 +1,95 @@
+/* global describe, it, expect, jasmine */

```

Comment:
you are going to totally not like me for this, but we are now writing our [specs](https://github.com/deepstreamIO/deepstream.io-e2e)  in cucumber (a)

On the bright side, its ALOT easier in jsland! I'm more than happy to help out with that if needed.